packs/feat-effects/effect-turn-aside-ambient-magic.json

```diff
@@ -0,0 +1,69 @@
+{
+    "_id": "uRwnbZ0xWrB5ZzZ9",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Turn Aside Ambient Magic",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Turn Aside Ambient Magic]</p>\n<p><span style=\"font-family:Roboto, sans-serif\">Your ostilli glows a faint yellow as it establishes a barrier. Choose acid, cold, electricity, fire, or sonic damage. Until the beginning of your next turn, you gain resistance against the chosen damage type equal to half your level.</span></p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of the Wild"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "Acid",
+                        "value": "acid"
+                    },
+                    {
+                        "label": "Cold",
+                        "value": "cold"
+                    },
+                    {
+                        "label": "Electricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "Fire",
+                        "value": "fire"
+                    },
+                    {
+                        "label": "Sonic",
+                        "value": "sonic"
+                    }
+                ],
+                "flag": "damage",
+                "key": "ChoiceSet",
+                "prompt": "Choose a damage type"
+            },
```

Comment:
Be sure to localize these. They all already have existing strings. The choices should all be `PF2E.Trait<name>` and the prompt will be under `PF2E.SpecificRule.Prompt.DamageType`